### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,50 +12,50 @@
...
       </div>
 
       <table class="game-field">
         <tbody>
           <tr class="field-row">
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
+            <td id="0-0" class="field-cell"></td>
```
Comment (mykhalenych) [inline src/index.html:24]: Adding ids for individual table cells may not be the most efficient way to manage cell data. Consider handling this in JavaScript with an array or object instead.

File: src/index.html [hunk 2]
```diff
@@ -12,50 +12,50 @@
         <h1>2048</h1>
         <div class="controls">
           <p class="info">
-            Score: <span class="game-score">0</span>
+            Score: <span id="game-score" class="game-score">0</span>
           </p>
-          <button class="button start">Start</button>
+          <button id="button-start" class="button start">Start</button>
```
Comment (mykhalenych) [inline src/index.html:17]: the same

File: src/index.html (hunk 1 above)
Comment (mykhalenych) [reply src/index.html:24]: Screenshot 2023-09-04 at 16 31 34

you can use .classList.add(`field-cell--${num}`) and .classList.replace

Comment (mykhalenych) [reply src/index.html:24]: you already do this way, delete all ids

Comment (MishaLehotskyi) [reply src/index.html:24]: cannot delete id's because it is main part my program's logic

**Review by maxim2310 — APPROVED** [review]
Good work, but next time add demolink to PR)

**Pull request closed without merging**